faiss/invlists/OnDiskInvertedLists.h

```diff
@@ -77,6 +77,8 @@ struct OnDiskInvertedLists : InvertedLists {
     size_t totsize;
     uint8_t* ptr;   // mmap base pointer
     bool read_only; /// are inverted lists mapped read-only
+    bool pre_mapped;// whether the content is already mmap'd before class creation
+    bool prefetch; // whether to prefetch lists
```

Comment:
Can we explain the significance of this better? Perhaps a better name more indicative of what it's for.